shared/src/main/java/uk/gov/di/authentication/shared/entity/NotificationType.java

```diff
@@ -3,5 +3,6 @@
 public enum NotificationType {
     VERIFY_EMAIL,
     VERIFY_PHONE_NUMBER,
-    MFA_SMS
+    MFA_SMS,
+    EMAIL_UPDATED
 }
```

Comment:
its probably worth separating these out to use its own for account management even if they are the same

Reply:
Have updated